packages/performance/wafer-map/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { waferMapTag } from '@ni/nimble-components/dist/esm/wafer-map';
+import type { WaferMapDie } from '@ni/nimble-components/dist/esm/wafer-map/types';
+
+document.addEventListener('DOMContentLoaded', main);
+
+async function waitAnimationFrame(): Promise<void> {
+    return new Promise(resolve => {
+        requestAnimationFrame(() => resolve());
+    });
+}
+
+async function runBenchmark(benchmark: string, dieCount: number): Promise<void> {
+    const waferMapDies = generateDies(dieCount);
+
+    const start = performance.now();
+    const waferMap = document.createElement(waferMapTag);
+    waferMap.dies = waferMapDies;
+    waferMap.colorScale = {
+        colors: ['red', 'orange', 'yellow', 'green'],
+        values: ['1', '33', '66', '100']
+    };
+    document.body.append(waferMap);
+    await waitAnimationFrame();
+    performance.measure(benchmark, { start });
+
+    document.body.removeChild(waferMap);
+}
+
+function main(): void {
+    void (async (): Promise<void> => {
+        document.querySelector<HTMLParagraphElement>('.running')!.hidden = false;
+        await runBenchmark('wafer_10K_dies', 10 ** 4);
+        await runBenchmark('wafer_100K_dies', 10 ** 5);
+        await runBenchmark('wafer_1M_dies', 10 ** 6);
```

Comment:
Made the names more descriptive / useful. When looking at the stats "small" does not tell you as much about the benchmark as "10k dies"